CMakeLists.txt

```diff
@@ -334,7 +337,7 @@ add_library(jsoncpp OBJECT third_party/jsoncpp/jsoncpp.cpp)
 
 # General packaging configuration
 set(CPACK_GENERATOR "DEB")
-set(CPACK_DEBIAN_PACKAGE_MAINTAINER "anton.gerasimov@here.com")
+set(CPACK_DEBIAN_PACKAGE_MAINTAINER "Patrick Vacek <patrick.vacek@here.com>")
```

Comment:
lol again

Reply:
Yes this should be the final version, the linter was complaining about missing name of maintainer...